cmdtests/sql_index_util.go

```diff
@@ -0,0 +1,55 @@
+// +build integration regression
+
+package cmdtests
+
+import (
+	"strings"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"gotest.tools/icmd"
+)
+
+// building/loading index takes some time, wait maximum 15s
+func IndexIsVisible(s commandSuite, table, name string) bool {
+	for i := 0; i < 15; i++ {
+		if hasIndex(s, table, name) {
+			return true
+		}
+		time.Sleep(time.Second)
+	}
+
+	return hasIndex(s, table, name)
+}
+
+func hasIndex(s commandSuite, table, name string) bool {
+	r := s.RunCommand("sql", "SHOW INDEX FROM "+table)
+	s.Require().NoError(r.Error, r.Combined())
+
+	// parse result and check that correct index was built and it is visiable
+	lines := strings.Split(r.Stdout(), "\n")
```

Comment:
Maybe we could add an example output. This way if the format changes in the future we can compare it and fix the code easier.

Reply:
I added a link instead of output itself (it's huge). The link contains version of mysql so it won't change.